docs/develop/api-reference/index.mdx

```diff
@@ -445,3 +450,7 @@ raise response.error if response.error?
   `}
   cli={`momento cache delete --key test-key --value test-value`}
 />
+
+### Collection data types (CDTs)
+
+Collection data types include [dictionaries](./dictionary-collections.md), [lists](./list-collections.md), and [sets](./set-collections.md).
```

Comment:
I haven't gotten a feel for the docs yet, but I'd typically think a section like this would have a definition of what collection data types are and underneath have a bulleted list with links to the data types

Example:

```
### Collection Data Types (CDT)

Collections may contain different types of structures depending on your use case. Supported data types are:

* [Dictionaries](./dictionary-collections.md)
* [Lists](./list-collections.md)
* [Sets]((./set-collections.md)
```
If this doesn't jive with what you already have, I'm happy to discuss pros and cons with an approach like this.

Reply:
I was trying to go quick to get something out, but FINE!! ;-P You're right and I added more information.